src/modules/layerManager/components/LayerItem.js

```diff
@@ -132,7 +132,7 @@ export class LayerItem extends AbstractContentPanel {
 
 		const openLayerInfoPanel = async () => {
 			const content = html`<ba-layerinfo-panel .geoResourceId=${this._layer.id}></ba-layerinfo-panel>`;
-			openModal(this._layer.label, content);
+			openModal(this._layer.geoResourceId, content);
```

Comment:
Here we have to use the label of the GeoResource instance as the title for the modal.
The label we get by:
```
const geoR = this._geoResourceService.byId(geoResourceId);
const label = geoR ? geoR.label : geoResourceId;

```
see `CatalogLeaf.js` line 43